spec/statement.dd

```diff
@@ -260,21 +260,21 @@ $(GNAME ElseStatement):
         initialized to the end of the $(I ThenStatement).)
 
 ---
-import std.regexp;
+import std.regex;
 ...
-if (auto m = std.regexp.search("abcdef", "b(c)d"))
+if (auto m = std.regex.matchAll("abcdef", "b(c)d"))
 {
-    writefln("[%s]", m.pre);      // prints [a]
-    writefln("[%s]", m.post);     // prints [ef]
-    writefln("[%s]", m.match(0)); // prints [bcd]
-    writefln("[%s]", m.match(1)); // prints [c]
-    writefln("[%s]", m.match(2)); // prints []
+    writefln("[%s]", m.pre);    // prints [a]
+    writefln("[%s]", m.post);   // prints [ef]
+    writefln("[%s]", m.hit[0]); // prints [b]
+    writefln("[%s]", m.hit[1]); // prints [c]
+    writefln("[%s]", m.hit[2]); // prints [d]
```

Comment:
> /tmp/temp_7F029904D1B0.d(12,27): Error: no [] operator overload for type RegexMatch!(string, ThompsonMatcher)

Ah, I think it should use `matchFirst`, not `matchAll`.

> Anyway does it really matter ?

Not really, no. But since if you're fixing it, why not make the example also make sense (i.e. be one that closer reflects a real-world situation)?